test/test_helper.rb

```diff
@@ -1,8 +1,29 @@
 require 'json'
+require 'yaml'
 require 'katello_attach_subscription'
 require 'minitest/autorun'
 
 def read_json_fixture(file)
   json = File.expand_path(File.join('..', 'fixtures', "#{file}.json"), __FILE__)
   JSON.parse(File.read(json))
 end
+
+def read_yaml_fixture(file)
+  yaml = File.expand_path(File.join('..', 'fixtures', "#{file}.yaml"), __FILE__)
+  YAML.load_file(yaml)
+end
+
+def getSubForHost(subs, host)
```

Comment:
to match the other names, can you please call this `get_sub_for_host`?
```suggestion
def get_sub_for_host(subs, host)
```